### PR TITLE
Fixes #10469: Make SLES12 RPMs depend on java8 headless

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/10469

### CODE REVIEW

File: rudder-jetty/SPECS/rudder-jetty.spec [hunk 1]
```diff
@@ -88,10 +88,13 @@ Requires: jre >= 1.8
 Requires: java-1.8.0-openjdk-headless
 %endif
 
+%if 0%{?suse_version} && 0%{?suse_version} == 1315 && 0%{?sle_version} && 0%{?sle_version} >= 120100
```
Comment (amousset) [inline rudder-jetty/SPECS/rudder-jetty.spec]: I think we can remove the "suse_version" checks.

Comment (armeniaca) [inline rudder-jetty/SPECS/rudder-jetty.spec]: Apparently SLES12 has:
```
# rpm --eval '%{suse_version}'
1315
```

SLES11 has:
```
# rpm --eval '%{suse_version}'
1110
```

Comment (armeniaca) [inline rudder-jetty/SPECS/rudder-jetty.spec]: But no sle_version:

SLE11
```
# rpm --eval '%{sle_version}'
%{sle_version}
```
SLES12
```
# rpm --eval '%{sle_version}'
120200
```

Comment (amousset) [inline rudder-jetty/SPECS/rudder-jetty.spec]: It is consistent with what is documented there: https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto

Comment (amousset) [inline rudder-jetty/SPECS/rudder-jetty.spec]: Here we are targeting SLES12SP1+, so `0%{?sle_version} && 0%{?sle_version} >= 120100` should be ok.

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**